php_phongo.c

```diff
@@ -2445,6 +2445,12 @@ PHP_MINIT_FUNCTION(mongodb)
 
 	/* Initialize libmongoc */
 	mongoc_init();
+
+#ifdef MONGOC_EXPERIMENTAL_FEATURES
```

Comment:
mongoc 1.4.0 and libbson 1.4.0 have experimental support for MongoDB 3.4 features that we are working on.
This experimental support is experimental until the first release of MongoDB 3.4 since they can be removed or changed at any time.
We have no intentions of actually supporting MongoDB 3.4 features in our 1.4.x series, but we are developing with them in mind and actively trying out MongoDB 3.4 things.
This is one of such feature. All of the MongoDB 3.4 experimental features and lumped together under this flag.

We decided to go this approach rather then branching as it simplified maintenance for us.


Note that this feature hasn't actually been committed to MongoDB master, yet :)

Its up to you if you want to leave this PR hanging here for a while.
The main point for @puppyofkosh was to ensure the mongoc implementation of this accounted for how the PHP & C++11 drivers.
See also https://github.com/mongodb/mongo-cxx-driver/pull/509